sympy/plotting/pygletplot/plot_window.py

```diff
@@ -85,7 +85,16 @@ def draw(self):
         if len(self.plot._functions.values()) == 0:
             self.drawing_first_object = True
 
-        for r in self.plot._functions.itervalues():
```

Comment:
Can this just use values()? It won't be an iterable in Python 2 but that usually isn't an issue.

Reply:
Maybe. I got an error for my python3 and googled for a 2/3-solution. I'll give it a try. (I also later noticed that there is a file for this type of functions, will move it there in case it won't work with values().)